packages/expo-in-app-purchases/ios/EXInAppPurchases/EXInAppPurchasesModule.m

```diff
@@ -167,6 +168,7 @@ -(void)handlePurchase:(SKProductsResponse *)response {
   }
   
   for (SKProduct *validProduct in response.products) {
+    if(!validProduct.localizedDescription) { continue; } // skip product with nil values - this can happen if it is in review "rejected" state
```

Comment:
```suggestion
    if (!validProduct.localizedDescription) { continue; } // skip product with nil values - this can happen if it is in review "rejected" state
```